### ISSUE TITLE
Panel chat: NVDA does not read the follow up command

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#26076]
Testing -186218

* Run `/createNotebook load csv to data frame`
* When the response finishes, NVDA starts to read the response
* 🐛 It ignores the follow up command to "Create Notebook"

<img width="354" alt="image" src="https://github-com/microsoft/vscode/assets/876920/b00fa483-36f9-4d75-9980-aba016a2f4a4">
